src/test/java/com/teragrep/pth_10/TeragrepTransformationTest.java

```diff
@@ -718,6 +720,33 @@ public void tgRegexExtractDefaultInputTest() {
                 );
     }
 
+    @Test
```

Comment:
For consistency, shouldn't this test also use @DisabledIfSystemProperty annotation like all the other tests?